src/com/lftechnology/java/training/alina/collections/generics/wildcards/CustomerUtil.java

```diff
@@ -0,0 +1,75 @@
+
+package com.lftechnology.java.training.alina.collections.generics.wildcards;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * CustomerUtil Class consists of comparisions of different customer
+ * 
+ * @author Alina Shakya <alinashakya@lftechnology.com>
+ * @param <T>
+ */
+public class CustomerUtil<T extends Customer> {
+
+	private T customer;
+
+	public CustomerUtil(T obj) {
+
+		this.setCustomer(obj);
+	}
+
+	public T getCustomer() {
+
+		return customer;
+	}
+
+	public void setCustomer(T customer) {
+
+		this.customer = customer;
+	}
+
+	public float getBill() {
+
+		return customer.getBill();
+	}
+
+	/**
+	 * Generic method to list customers
+	 * 
+	 * @param buttonList
+	 * @return customer {@link List} lists of customers
+	 * @author Alina Shakya <alinashakya@lftechnology.com>
+	 */
+	public static List<CustomerUtil<?>> getCustomers(
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove usage of generic wildcard type. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1452)